processing file asmFiles/test.beqtaken.asm (1 of 1)
processing file asmFiles/test.beqnottaken.asm (1 of 24)
processing file asmFiles/test.beqtaken.asm (2 of 24)
processing file asmFiles/test.bgenottaken.asm (3 of 24)
processing file asmFiles/test.bgetaken.asm (4 of 24)
processing file asmFiles/test.bgeunottaken.asm (5 of 24)
--- memsim.hex	2025-02-18 12:27:29.068077658 -0500
+++ memcpu.hex	2025-02-18 12:27:29.052077048 -0500
@@ -2,6 +2,5 @@
 :040001000030611357
 :040002000020F46383
 :04000300FFFFFFFFFD
-:04000400003126A3FE
 :04000500FFFFFFFFFB
 :00000001FF
processing file asmFiles/test.bgeutaken.asm (6 of 24)
--- memsim.hex	2025-02-18 12:27:30.827144661 -0500
+++ memcpu.hex	2025-02-18 12:27:30.809143975 -0500
@@ -2,5 +2,6 @@
 :040001000030611357
 :040002000020F46383
 :04000300FFFFFFFFFD
+:04000400003126A3FE
 :04000500FFFFFFFFFB
 :00000001FF
processing file asmFiles/test.bltnottaken.asm (7 of 24)
processing file asmFiles/test.blttaken.asm (8 of 24)
--- memsim.hex	2025-02-18 12:27:34.327277981 -0500
+++ memcpu.hex	2025-02-18 12:27:34.310277334 -0500
@@ -2,5 +2,6 @@
 :040001000060611327
 :040002000020C463B3
 :04000300FFFFFFFFFD
+:04000400003125237F
 :04000500FFFFFFFFFB
 :00000001FF
processing file asmFiles/test.bltunottaken.asm (9 of 24)
--- memsim.hex	2025-02-18 12:27:36.098345441 -0500
+++ memcpu.hex	2025-02-18 12:27:36.077344641 -0500
@@ -1,7 +1,6 @@
 :0400000000306093D9
 :040001000020611367
 :040002000020E46393
-:04000300FFFFFFFFFD
 :04000400003126237E
 :04000500FFFFFFFFFB
 :00000001FF
processing file asmFiles/test.bltutaken.asm (10 of 24)
processing file asmFiles/test.bnenottaken.asm (11 of 24)
processing file asmFiles/test.bnetaken.asm (12 of 24)
processing file asmFiles/test.branchjump1.asm (13 of 24)
--- memsim.hex	2025-02-18 12:27:43.191615625 -0500
+++ memcpu.hex	2025-02-18 12:27:43.173614939 -0500
@@ -16,6 +16,5 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
-:0400220000000024B6
-:040024000000003C9C
+:0400240000000024B4
 :00000001FF
processing file asmFiles/test.controlflowhazard.asm (14 of 24)
processing file asmFiles/test.datahazard.asm (15 of 24)
--- memsim.hex	2025-02-18 12:27:46.678748450 -0500
+++ memcpu.hex	2025-02-18 12:27:46.658747688 -0500
@@ -1,6 +1,4 @@
 :0400000000506193B8
-:0400010000000005F6
-:040002000000000AF0
 :040003000040228314
 :040004000032833310
 :040005000060242350
processing file asmFiles/test.forwardinghazard.asm (16 of 24)
--- memsim.hex	2025-02-18 12:27:46.678748450 -0500
+++ memcpu.hex	2025-02-18 12:27:48.445815758 -0500
@@ -1,8 +1,9 @@
 :0400000000506193B8
-:0400010000000005F6
-:040002000000000AF0
-:040003000040228314
-:040004000032833310
-:040005000060242350
-:04000600FFFFFFFFFA
+:040001000030262382
+:0400020000000013E7
+:040004000040611344
+:040005000030621352
+:04000600005203336E
+:04000700006026234C
+:04000800FFFFFFFFF8
 :00000001FF
processing file asmFiles/test.halt.asm (17 of 24)
processing file asmFiles/test.jal.asm (18 of 24)
--- memsim.hex	2025-02-18 12:27:52.001951211 -0500
+++ memcpu.hex	2025-02-18 12:27:51.984950564 -0500
@@ -2,5 +2,6 @@
 :0400010000406193C7
 :04000200008000EF8B
 :04000300FFFFFFFFFD
+:040004000041A623EE
 :04000500FFFFFFFFFB
 :00000001FF
processing file asmFiles/test.jalr.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
--- memsim.hex	2025-02-18 12:28:00.726283522 -0500
+++ memcpu.hex	2025-02-18 12:28:00.708282836 -0500
@@ -27,12 +27,12 @@
 :04001A00007AAE2397
 :04001B0003CAA02351
 :04001C00FFFFFFFFE4
-:0401C10000001261C7
+:0401C10000000261D7
 :0401C2000000000930
-:0401C30000010A5AD3
+:0401C30000000A5AD4
 :0401C40000000740F0
-:0401C500FFFFDA70EE
-:0401C600000037F806
-:0401C7000000D1560D
-:0401C800000D269070
+:0401C500FFFFFA70CE
+:0401C600000007F836
+:0401C70000000156DD
+:0401C800000026907D
 :00000001FF
processing file asmFiles/test.structuralhazard.asm (24 of 24)
